fentoboardimage/Coordinates.py

```diff
@@ -4,33 +4,35 @@
 from .Utils import flippedCheck, indicesToSquare
 
 
+def get_size(font, s):
+    left, top, right, bottom = font.getbbox(s)
+    width, height = right - left, bottom
```

Comment:
nit: Is top guaranteed to be 0? I'm wondering if height should be `bottom - top`? Looks like the font's origin [sometimes can be non-zero.](https://stackoverflow.com/a/76234689)

Reply:
Update: looks like the old method had an issue that ignored the top property, so this is accurate to the old implementation.